docs/visualization/helper_api/path_verify.md

```diff
@@ -1,8 +1,8 @@
 # Error detection using `path.verify()`
 
 CARET requires some setup and configuration that users should do.
-`path.verify()` method can check the settings are completed.
-If any error or warning occurs, message from `path.verify()` give you some instruction to tackle it.
+The function `path.verify()` can check that the settings are completed.
+If there are some problems of settings, this function shows hints to resolve them.
```

Comment:
I think links to configuration page is needed.
https://tier4.github.io/CARET_doc/pr-98/configuration/intra_node_data_path/

Reply:
@keita1523 

BTW, `path.verify` is described in configuration too.
https://tier4.github.io/CARET_doc/pr-98/configuration/intra_node_data_path/

Are there any reason to write the description into visualization?

Reply:
I merged it.